### PR TITLE
adding tags to keyvault

### PR DESCRIPTION
**Pull request by zikalino**
##### SUMMARY
Adding missing tags support

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
azure_rm_keyvault

##### ANSIBLE VERSION
2.7

### CODE REVIEW

File: test/integration/targets/azure_rm_keyvault/tasks/main.yml [hunk 1]
```diff
@@ -82,6 +82,8 @@
         keys:
           - get
           - list
+    tags:
+      aaa: bbb
   register: output
 - name: Assert the state has changed
   assert:
```
Comment (yuwzho) [inline test/integration/targets/azure_rm_keyvault/tasks/main.yml:89]: add assert here

Comment (zikalino) [reply test/integration/targets/azure_rm_keyvault/tasks/main.yml:89]: it's not possible to tests like this. this module doesn't return the whole structure (following new rules) and no facts module available yet. we could possibly use rest module for test?

Comment (zikalino) [reply test/integration/targets/azure_rm_keyvault/tasks/main.yml:89]: ok, added test for tags using rest modules

**Comment by yuwzho** [comment]
shipit

File: test/integration/targets/azure_rm_keyvault/tasks/main.yml [hunk 2]
```diff
@@ -82,12 +82,27 @@
...
     that:
       - output.changed == true
 
+- name: Use REST API to retrieve keyvault (as facts not available yet)
```
Comment (yuwzho) [inline test/integration/targets/azure_rm_keyvault/tasks/main.yml:93]: doesn't show in the module return value?

Comment (zikalino) [reply test/integration/targets/azure_rm_keyvault/tasks/main.yml:93]: no, this is one of the new modules, and the rule is not to return input information.

**Review by yuwzho — APPROVED** [review]

**Comment by yuwzho** [comment]
shipit

**Review by yungezz — APPROVED** [review]

**Pull request merged**